docs/rules/README.md

```diff
@@ -276,42 +277,42 @@ The following rules extend the rules provided by ESLint itself and apply them to
 
 | Rule ID | Description |    |    |
 |:--------|:------------|:--:|:--:|
-| [vue/array-bracket-newline](./array-bracket-newline.md) | enforce linebreaks after opening and before closing array brackets in `<template>` | :wrench: | :lipstick: |
```

Comment:
Please run these commands in your branch:

```bash
git remote add upstream git@github.com:vuejs/eslint-plugin-vue.git
git fetch upstream
git merge upstream/master
npm run update
git commit -m "update docs"
git push
```

Reply:
Thank you for your patience

Reply:
fetch and run update，but no file files have been changed

Reply:
Note that you need to pull the changes from the `upstream` repository (i.e. https://github.com/vuejs/eslint-plugin-vue). Did you do that?

Reply:
I performed the following actions

Reply:
Ah, the changes are actually fine then :)

I added https://github.com/vuejs/eslint-plugin-vue/commit/edb7ec5384ff55ca064aa5f1ee2cd104a094a807 to the `master` branch and updated your branch, so the diff is fine again :)

Reply:
ok